wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -775,7 +775,7 @@ public final class MLSService: MLSServiceInterface {
                     customConfiguration: .init(keyRotationSpan: nil, wirePolicy: nil)
                 )
             }
-            let groupID = MLSGroupID(groupIDData)
+            let groupID = MLSGroupID(groupIDData.id)
```

Comment:
Shouldn't we rename `groupIDData` to welcomeBundle?

Reply:
Done